mitosheet/src/components/taskpanes/Merge/MergeTaskpane.tsx

```diff
@@ -56,10 +57,26 @@ export type MergeTaskpaneProps = {
     setUIState: React.Dispatch<React.SetStateAction<UIState>>;
     mitoAPI: MitoAPI,
     analysisData: AnalysisData;
+    /* 
+        These props are only defined if we are editing a merge
+        that already exists, and these are what we then default it to
+    */
+    existingMergeParams?: MergeParams;
+    destinationSheetIndex?: number;
 };
 
 
-export const getDefaultMergeParams = (sheetDataArray: SheetData[], _sheetIndexOne: number, _sheetIndexTwo?: number, previousParams?: MergeParams): MergeParams | undefined => {
+export const getDefaultMergeParams = (
+    sheetDataArray: SheetData[], 
+    _sheetIndexOne: number, 
+    _sheetIndexTwo?: number, 
+    previousParams?: MergeParams, // When we change the merge params, we use the previous params to handle cases like switching input sheets
+    existingMergeParams?: MergeParams // If we are editing a merge, this is the configuration to start with
```

Comment:
This is really a code smell, these two variables are confusing. Can we refactor this into two different functions? I think there can be a getDefaultMergeParams and a getUpdatedMergeParams function (if we need).